documentation/overrides/partials/banner.html

```diff
@@ -0,0 +1,9 @@
+    <p>For help, click the link below to get free database assistance or contact our experts for personalized support.</p>
+
+    <div class="actions">
+
+    <a href="https://docs.percona.com/percona-monitoring-and-management/3/get-help.html" style="color: var(--md-typeset-a-color);">Get help from Percona</a>
```

Comment:
That [page](https://docs.percona.com/percona-monitoring-and-management/3/get-help.html) doesn't seem to exist yet, does it?